pages/1.13/security/ent/secure-compute-profiles/index.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: layout.pug
+navigationTitle:  Working with secure computing profiles
+title: Working with secure computing profiles
+menuWeight: 31
+excerpt: Describes how you can work with Linux secure computing (seccomp) profiles 
+enterprise: true
+---
+<!-- The source repository for this topic is https://github.com/dcos/dcos-docs-site -->
+The Linux kernel provides a native computer security facility that you can configure to control the system-level operations that can be performed. This native security facility, the secure computing mode (Seccomp), is included by default in most distributions of the Linux kernel. The secure computing mode (seccomp) provides system call isolation that enables programs to run safely without compromising the systems where the programs run.
+
+In a distributed network environment, you can use the secure computing mode (Seccomp) to isolate applications running inside of containers from critical system resources and programs. The secure computing mode isolates containers and tasks by enabling you to restrict access to certain system calls through a secure computing profile. By configuring a seccomp profile, you can specify a list of restricted and permitted system calls for containers launched by the DC/OS Universal Container Runtime (UCR).
+
+The seccomp profile identifies:
+- individual restricted or permitted system calls, such as `exit()`, `sigreturn()`, `read()`, or `write()`.
+- filters for system calls by `syscall` arguments, similar to `prctl(PR_SET_SECCOMP, SECCOMP_MODE_FILTER, filter)` and `prctl(PR_SET_NO_NEW_PRIVS, 1, 0, 0, 0)`.
```

Comment:
replace `similar to` with
```
for example you can restrict system call `prctl` in case it is called with the following arguments `prctl(PR_SET_SECCOMP, SECCOMP_MODE_FILTER, filter)` or `prctl(PR_SET_NO_NEW_PRIVS, 1, 0, 0, 0)`
```

Reply:
That's so much better than my garbled version. Fixed.